apps/docs/src/app/core/component-docs/feed-input/examples/feed-input-disabled-example/feed-input-disabled-example.component.html

```diff
@@ -0,0 +1,26 @@
+<fd-feed-input
+    [disabled]="true"
+    aria-label="Feed input example">
+
+    <fd-avatar
+        class="fd-feed-input__thumb"
```

Comment:
Why do you have `class` here? Shouldn't you have a directive that will bring this class?

Reply:
Added directive